linera-execution/tests/wasm.rs

```diff
@@ -26,10 +26,10 @@ use test_case::test_case;
 /// called correctly and consume the expected amount of fuel.
 ///
 /// To update the bytecode files, run `linera-execution/update_wasm_fixtures.sh`.
-#[cfg_attr(feature = "wasmer", test_case(WasmRuntime::Wasmer, 28_496, ExecutionRuntimeConfig::Synchronous; "wasmer"))]
-#[cfg_attr(feature = "wasmer", test_case(WasmRuntime::WasmerWithSanitizer, 28_785, ExecutionRuntimeConfig::Synchronous; "wasmer_with_sanitizer"))]
-#[cfg_attr(feature = "wasmtime", test_case(WasmRuntime::Wasmtime, 28_785, ExecutionRuntimeConfig::Synchronous; "wasmtime"))]
-#[cfg_attr(feature = "wasmtime", test_case(WasmRuntime::WasmtimeWithSanitizer, 28_785, ExecutionRuntimeConfig::Synchronous; "wasmtime_with_sanitizer"))]
+#[cfg_attr(feature = "wasmer", test_case(WasmRuntime::Wasmer, 29_140, ExecutionRuntimeConfig::Synchronous; "wasmer"))]
```

Comment:
Fuel consumption is going up??

Reply:
I found that weird as well. But I can't rule out that it's just a compilation issue. I can't remember if I was the last one to update the fixtures, so someone else might have gotten a more optimized binary. I can run a before and after test and report back.

Reply:
Okay, I updated the test fixtures with the commit on `main` before this PR was merged, and I got the fuel consumption values of 29,152 for Wasmer without being sanitized and 29,457 for all others. So with this PR there's a slight improvement of 12 fuel units to 29,140 and 29,445 respectively.